<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sniffer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/myshark.ico</normaloff>:/images/myshark.ico</iconset>
  </property>
  <widget class="QWidget" name="Widget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QSplitter" name="vSplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="text">
         <string>stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="restartButton">
        <property name="text">
         <string>restart</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="text">
         <string>clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reloadButton">
        <property name="text">
         <string>reload</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="arpattacker">
        <property name="text">
         <string>arp attacker</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>600</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,0,0">
      <item>
       <widget class="QPushButton" name="setFilterButton">
        <property name="text">
         <string>set</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>filter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="filterTextLabel">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>150</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="chooseAdapterButton">
        <property name="text">
         <string>choose adapter</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>network adapter</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="adapterName">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>NULL</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionabout"/>
   </widget>
   <widget class="QMenu" name="get_menu">
    <property name="title">
     <string>捕获(&amp;C)</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="actionReget"/>
   </widget>
   <widget class="QMenu" name="tool_menu">
    <property name="title">
     <string>工具(&amp;T)</string>
    </property>
    <addaction name="actionstartARP"/>
    <addaction name="actionstopARP"/>
   </widget>
   <widget class="QMenu" name="view_menu">
    <property name="title">
     <string>视图(&amp;V)</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>解析方式</string>
     </property>
     <addaction name="actionViewip"/>
     <addaction name="actionViewmac"/>
    </widget>
    <addaction name="menu"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="get_menu"/>
   <addaction name="view_menu"/>
   <addaction name="tool_menu"/>
   <addaction name="help_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStart">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
   <property name="toolTip">
    <string>开始捕获分组</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>停止</string>
   </property>
   <property name="toolTip">
    <string>停止捕获分组</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionReget">
   <property name="text">
    <string>重新开始</string>
   </property>
   <property name="toolTip">
    <string>重新开始当前捕获</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存捕获文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="toolTip">
    <string>关闭捕获文件</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>重新加载</string>
   </property>
   <property name="toolTip">
    <string>重新加载文件</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionstartARP">
   <property name="text">
    <string>配置ARP欺骗</string>
   </property>
  </action>
  <action name="actionstopARP">
   <property name="text">
    <string>停止ARP欺骗</string>
   </property>
  </action>
  <action name="actionViewip">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>解析IP</string>
   </property>
  </action>
  <action name="actionViewmac">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>解析MAC</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
